natlas-server/README.md

```diff
@@ -12,13 +12,27 @@ Backing services in the Natlas server are defined via environment configs. They
 
 ## Installation (Production)
 
-Production-ready Natlas docker containers are [available on dockerhub](https://hub.docker.com/r/natlas/server).
-
 Before you launch your Natlas server, ensure that you've already setup an [Elasticsearch](/README.md#Elasticsearch) cluster that your container will be able to reach. You'll then need to create your environment config file to mount into your container. Finally, choose whether you want to store media on the local filesystem in a mounted directory or if you want to use a docker volume. For simplicity, I'll be using a local filesystem mount.
 
+### Downloading Natlas Server
+
+Production-ready Natlas docker containers are [available on dockerhub](https://hub.docker.com/r/natlas/server). If you'd like to grab the latest snapshot from `main`, it's as simple as this:
```

Comment:
`main` -> `latest`

Reply:
Main is the git branch, which is what I meant. I can see the confusion though.

Reply:
I actually want to remove this I think. It's not relevant to this change set and I'd like to start maintaining the authoritative docs in only one place, the docs website

Reply:
Leaving it here for now but I made it clearer. I think a separate PR is in order for making sure everything has been transferred to the docs site and then we can start to reduce the complexity of the in-repo readmes by pointing them over to the docs pages where applicable.